RecoEgamma/Examples/plugins/BuildFile.xml

```diff
@@ -40,11 +39,13 @@
 <use   name="DataFormats/TrackReco"/>
 <use   name="DataFormats/TrajectoryState"/>
 <use   name="DataFormats/VertexReco"/>
+<use   name="DQMServices/Core"/>
 <use   name="SimDataFormats/CrossingFrame"/>
 <use   name="SimDataFormats/GeneratorProducts"/>
 <use   name="SimDataFormats/Track"/>
 <use   name="SimDataFormats/TrackingAnalysis"/>
 <use   name="SimDataFormats/Vertex"/>
+<use   name="classlib"/>
```

Comment:
I do not see any classlib/ includes in RecoEgamma.
does this dependence come through some other packages headers?
Usually that dependence should bring in everything else and there is no need to unroll to the full dep list

Reply:
Good catch, you are right. I was adding it because it didn't compile, and then I added `DQMStore/Core` because it still didn't compile. Now I understand that only adding the latter is enough, so I will amend the last commit.